src/shared/defaultAwsContextCommands.ts

```diff
@@ -72,12 +73,13 @@ export class DefaultAWSContextCommands {
             const profileName: string | undefined = await this.promptAndCreateNewCredentialsFile()
 
             if (profileName) {
-                await this.loginManager.login({ passive: false, providerId: fromString(profileName) })
+                // TODO: change this once we figure out what profile types we should have
+                const profileNameWithType: string = SharedCredentialsProvider.getCredentialsType() + ':' + profileName
```

Comment:
A bit more formal and obvious/self-documenting, to create a `CredentialsProviderId` object directly. E.g. (untested):

```suggestion
                const c: CredentialsProviderId = {
                    credentialType: SharedCredentialsProvider.getCredentialsType(),
                    credentialTypeId: profileName
                }
                await this.loginManager.login({ passive: false, providerId: c })
```

Reply:
You'd think that, but `promptAndCreateNewCredentialsFile` was made before `CredentialsProviderId` was introduced. The function returns only the profile name provided by the user, it makes no assumptions as to what kind of credentials it is.